src/datasource/plugin.json

```
@@ -7,7 +7,7 @@
   "metrics": true,
   "backend": true,
   "alerting": true,
-  "executable": "../gpx_twinmaker_app",
+  "executable": "gpx_twinmaker_app",
```

It might be a nice idea to also update this to be more explicit 
```suggestion
  "executable": "gpx_twinmaker_datasource",
```

---

Hm - not sure I follow here. Twinmaker is an app, and I see some other apps using this format. Happy to follow your guidance, just want to understand the reasoning :)

---

This is because the datasource is the actual backend plugin piece. Technically you can have an app plugin with a backend, but in this case it's an app that bundles/includes a backend datasource. Just something I noticed, but just a small detail in reality 😄

---

Ah! TIL. Let's go ahead and change it then.